src/Test/contlinear.jl

```diff
@@ -171,8 +171,16 @@ function linear1test(model::MOI.ModelLike, config::TestConfig)
     vc3 = MOI.addconstraint!(model, MOI.SingleVariable(v[3]), MOI.GreaterThan(0.0))
     @test MOI.get(model, MOI.NumberOfConstraints{MOI.SingleVariable,MOI.GreaterThan{Float64}}()) == 3
 
-    @test MOI.canmodifyconstraint(model, c, MOI.ScalarCoefficientChange{Float64})
-    MOI.modifyconstraint!(model, c, MOI.ScalarCoefficientChange{Float64}(z, 1.0))
+    if config.modify_lhs
+        @test MOI.canmodifyconstraint(model, c, MOI.ScalarCoefficientChange{Float64})
+        MOI.modifyconstraint!(model, c, MOI.ScalarCoefficientChange{Float64}(z, 1.0))
+    else
```

Comment:
Should we add in the else:
```julia
@test !MOI.canmodifyconstraint(model, c, MOI.ScalarCoefficientChange{Float64})
```

Reply:
Once use case for this is when your code for `modifyconstraint!` seems to contain a but and you want to disable it to test if the rest works. For this use case, `canmodifyconstraint` returns `true`.

Reply:
ok